zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java

```diff
@@ -61,6 +61,7 @@
   Date dateUpdated;
   private Map<String, Object> config; // paragraph configs like isOpen, colWidth, etc
   public final GUI settings;          // form and parameter settings
+  private Map<String, ParagraphRuntimeInfo> runtimeInfos;
```

Comment:
Looks like `excludeFields` callback  of `ExcluionStrategy` is not triggered for field with `transient` modifiers.
http://grepcode.com/file/repo1.maven.org/maven2/com.google.code.gson/gson/2.3/com/google/gson/internal/Excluder.java#150

Should i remove `transient` modifier , and add the excluson strategy for all `NotebookRepo` ?? @Leemoonsoo